package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "watson-developer-cloud",
+  "name": "node-sdk",
```

Comment:
This change would prevent us from being able to update the library in npm, please undo it.